<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>天工艺彩客户端V1.0</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/gitwork/git/pyside2/src/project2/designerUI/Demo2/iconServer.ico</normaloff>C:/gitwork/git/pyside2/src/project2/designerUI/Demo2/iconServer.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#Form {
	background: url(:/resourse/Project/project2/src/resourse/background.png);  /* border-image */
}</string>
  </property>
  <widget class="QWidget" name="Login" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>311</width>
     <height>401</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>1</number>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget #Form {
    background-color: rgb(85, 85, 127);
    
	
	z-index: 2;
	/* color: white; */	
}

QWidget #Login {
	border-radius: 20px;
	background-color: #323232;
	font-family: Arial, Helvetica; 
}


QLabel, QPushButton, QCheckBox {
	color: white;
}
QLineEdit {
	height: 30px;
}

QLabel #Label_SignIn {
	font-size: 20px;
	font-weight: bold;
}

QPushButton#btn_close:pressed, #btn_min:pressed {
	border: 2px solid skyblue;
	border-radius: 10px;
}

QPushbutton #btn_SignIn {
	
}




</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>15</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/Administrator/.designer/resourse/logo.ico</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>291</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="userBox" stretch="1,2,1,18,1">
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="userLabel">
         <property name="pixmap">
          <pixmap>C:/Users/Administrator/Desktop/images/user4.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="openExternalLinks">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="userLine">
         <property name="styleSheet">
          <string notr="true">border-radius: 10px;
background-color: rgb(179, 179, 179);
font-size:14px;

</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>username</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="userSpaceRight">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="passwordBox" stretch="1,2,1,18,1">
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="passwordLabel">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>C:/Users/Administrator/Desktop/images/password.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="passwordLine">
         <property name="styleSheet">
          <string notr="true">border-radius: 10px;
background-color: rgb(179, 179, 179);
font-size:14px;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>password</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="passwordSpacerRight">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,8,12">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="Label_SignIn">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Sign in</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_SignIn">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="tabletTracking">
          <bool>false</bool>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="toolTipDuration">
          <number>4</number>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">color: white;
border-radius: 25px; 

</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>C:/Users/Administrator/.designer/resourse/login.png</normaloff>C:/Users/Administrator/.designer/resourse/login.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
         <property name="autoRepeatInterval">
          <number>100</number>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btn_SignUp">
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Comic Sans MS&quot;;
text-decoration: underline;
color: rgb(179, 179, 179);
background-color: rgb(85, 85, 127); 
</string>
         </property>
         <property name="text">
          <string>Sign up</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_Forgot">
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Comic Sans MS&quot;;
text-decoration: underline;
color: rgb(179, 179, 179);
background-color: rgb(85, 85, 127); </string>
         </property>
         <property name="text">
          <string>Forgot Passwords</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>11</y>
      <width>251</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>天工艺彩</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_min">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>C:/Users/Administrator/.designer/resourse/min.png</normaloff>C:/Users/Administrator/.designer/resourse/min.png</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_close">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>C:/Users/Administrator/.designer/resourse/close2.png</normaloff>C:/Users/Administrator/.designer/resourse/close2.png</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
